LLama/LLamaInteractExecutor.cs

```diff
@@ -25,7 +26,8 @@ public class InteractiveExecutor : StatefulExecutorBase
         /// 
         /// </summary>
         /// <param name="context"></param>
-        public InteractiveExecutor(LLamaContext context) : base(context)
+        /// <param name="logger"></param>
+        public InteractiveExecutor(LLamaContext context, ILogger logger = null!) : base(context, logger)
```

Comment:
```suggestion
        public InteractiveExecutor(LLamaContext context, ILogger? logger = null) : base(context, logger)
```

Reply:
You may need to change it to `ILogger?` in the base constructor as well

Reply:
Ahh yeah sorry, I dont use that nullable nonsense, Ill change my VS settings for LLamaSharp so I dont keep doing it :p